src/main/java/com/jadventure/game/prompts/CommandCollection.java

```diff
@@ -291,12 +282,12 @@ public void command_teleport(String arg) {
         }
     }
 
-    @Command(command="backpack", aliases="", description="Opens the backpack debug menu.", debug=true)
+    @Command(command="backpack", aliases={""}, description="Opens the backpack debug menu.", debug=true)
     public void command_backpack(String arg) {
         new BackpackDebugPrompt(player);
     }
-    
-    @Command(command="talk", aliases="t", description="Talks to a character.", debug=false)
+
+    @Command(command="talk", aliases={"talk", "speakto"}, description="Talks to a character.", debug=false)
```

Comment:
This drops the existing `t` alias

Reply:
Ah sorry! Mistake on my part. Will add the existing alias and clean up everything as and re-request a review for the PR